src/components/StyledProgress/StyledProgress.tsx

```diff
@@ -85,15 +85,21 @@ const StyledProgress = ({
         <>
           <ProgressBar
             style={{
-              width: percentageFormat(secondary, locale).replace('\xa0', ''),
+              width: `calc(${percentageFormat(secondary, locale).replace(
```

Comment:
I didn't notice this before, but I don't think we should be using a localized percentage format for the CSS width. Different locales use different separators or even put the percent sign first, which will result in invalid CSS syntax. I think we should just do `(secondary / 100).toFixed() + '%'` since we want the percentage in a specific format. Thanks for the `- 4px` fix though!

Reply:
I've tried many different languages and locales, and all those I tried worked. I'm using the same method to calculate the percentage as we have been doing. I haven't
![Screenshot 2024-12-13 at 10 40 00 AM](https://github.com/user-attachments/assets/1d370539-5855-4273-813d-6564bea461b0)
 hear of the progress bars not working for users.
![Screenshot 2024-12-13 at 10 44 22 AM](https://github.com/user-attachments/assets/6f806b2b-1585-4b97-b4d7-99de9c8e0c3b)

Reply:
Are you changing the language or the locale? On my account, I'm seeing a difference when switching between `en-US` and `ar`.

Here it is with American locale:

<img width="781" alt="Screenshot 2024-12-13 at 11 26 33 AM" src="https://github.com/user-attachments/assets/d75aa068-83b3-45ea-8cf3-0fa7e6230c0d" />

Here it is with Arabic locale:

<img width="785" alt="Screenshot 2024-12-13 at 11 26 15 AM" src="https://github.com/user-attachments/assets/11598dc9-bbaf-4bd3-8641-d9beb14e35fd" />

But maybe I'm missing something.

Reply:
Weird, maybe I needed to do a hard refresh to flush the cache

Reply:
Good catch